composer.json

```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": ">=2.0,<3.0",
+        "symfony/framework-bundle": ">=2.0,<=4.0",
```

Comment:
I suggest to use it like ~2.0|~3.0, the suggestion by reminec would not allow to use it , for example, with Symfony 2.7.